doc/release-notes-6229.md

```diff
@@ -0,0 +1,3 @@
+RPC changes
+-----------
+
```

Comment:
?

Reply:
Yup :) also forgot to archive old release notes. See latest